### PR TITLE
Include SearchInput Component

### PR DESCRIPTION
**Pull request by mirhamasala**
Style `SearchInput` component per [Figma Search Component Design](https://www.figma.com/file/0pBfQs5tKI6bx5ypPIfVRA/FIL.org-V4-Prototypes?type=design&node-id=1231-22293&mode=design&t=wKxwbAmsrvl62Vxv-4)

Screenshot 2024-04-11 at 15 42 37

### CODE REVIEW

File: src/app/_components/SearchInput.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  return (
+    <div className="w-full max-w-lg lg:max-w-xs">
+      <label htmlFor="search" className="sr-only">
+        Search
+      </label>
+      <div className="relative">
+        <input
+          id="search"
+          name="search"
+          className="peer block w-full rounded-lg border border-brand-300 bg-brand-800 py-3 pl-10 pr-3 placeholder:text-brand-300 hover:border-brand-400 placeholder:hover:text-brand-400 focus:border-transparent focus:text-brand-100 focus:outline-2 focus:outline-brand-100 placeholder:focus:text-brand-100"
```
Comment (CharlyMartin) [inline src/app/_components/SearchInput.tsx:20]: I didn't know about the `peer` modifier, nice one 😎

**Review by CharlyMartin — COMMENTED** [review]
Nice one @mirhamasala!

On Firefox, when the search element is in `focus` / `focus-visible` state, there's a weird double outline situation. On Chrome is all good.

CleanShot 2024-04-16 at 14 27 27@2x

**Comment by mirhamasala** [comment]
>On Firefox, when the search element is in focus / focus-visible state, there's a weird double outline situation. On Chrome is all good.

@CharlyMartin Thank you for noticing. `outline` doesn't seem to agree with Firefox on inputs - Should be fixed with `ring`.

**Comment by CharlyMartin** [comment]
Looks solid, good to merge 🙂

**Review by CharlyMartin — APPROVED** [review]

**Pull request merged**